controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -299,7 +261,10 @@ public void deleteStream(final String scopeName, final String streamName, final
         long traceId = LoggerHelpers.traceEnter(log, "deleteStream");
 
         try {
-            authenticateAuthorize(scopeName + "/" + streamName, READ_UPDATE);
+            restAuthHelper.authenticateAuthorize(
+                    getAuthorizationHeader(),
+                    AuthResourceRepresentation.ofStreamInScope(scopeName, streamName),
+                    READ_UPDATE);
```

Comment:
I thought we agreed that stream deletion was an action taken on the parent scope.